tests/Handler/StreamHandlerTest.php

```diff
@@ -270,6 +270,7 @@ public function testAddsProxy()
     public function testAddsProxyByProtocol()
     {
         $url = str_replace('http', 'tcp', Server::$url);
+        $url = rtrim($url, '/');
```

Comment:
Can you add a quick not why we are doing this? Just to make sure we find it. Thanks